docs/src/practices/tracking_issues.md

```diff
@@ -0,0 +1,55 @@
+# Tracking issues
+
+`nearcore` uses tracking issues on github to make it easier to follow progress
+on individual projects.
+
+## Purpose
+
+Different types of audiences want to track progress on feature development on `nearcore`:
+- Other engineers contributing to the feature development but who may have
+  missed some progress while they were unavailable.
+- Pagoda management who would like to have an overview of which teams are
+  working on what features and how much work is left to be done.
+- The general NEAR community who are interested in development of new features
+  in the protocol.
+
+Using tracking issues can help the above audience come up to speed with projects
+quickly and makes it easy for engineers working on projects to share updates.
+
+## Background
+
+The idea of tracking issues is also used to track project work in the Rust
+language.  See [this
+post](https://internals.rust-lang.org/t/how-the-rust-issue-tracker-works/3951)
+for a rough description and
+[these](https://github.com/rust-lang/rust/issues/101840)
+[issues](https://github.com/rust-lang/rust/issues/100717) for how they are used
+in Rust.
+
+## Guidelines
+
+- There should be one tracking issue per project.  Discretion can be used to
```

Comment:
Let's add a leading paragraph here, a-la: 


The main idea behind a tracking issue is to serve as a point of coordination. 
racking issue should be the single thing a new contributor needs to know to get fully up-to-speed with the state of a project. 

So, the main idea is to **link** things like design docs, todo-lists of susbissues, existing implementation PRs, etc.

Reply:
Oh, let's also link an example of a tracking issue from our repo. https://github.com/near/nearcore/issues/7670 would be a good example I think.

We could also add a githbub template for tracking issues, but I think there'll be little value in that: the main thing we want to get out of this process is the *culture* of openly tracking work in a collaboration-friendly way, not a specific form of a checkbox.